charts/lightstepsatellite/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.10
```

Comment:
Do we need to update the version in [this file](https://github.com/lightstep/lightstep-microsatellite-helm-chart/blob/d8e12c1306fda59db0a2a20323c5d5865d73e50b/charts/lightstepsatellite/README.md#L9)?

Reply:
Good eye, it seems version is embedded there for the badge links too...I'll update